### ISSUE TITLE
Parse Error: Expected '[' in Screen Capturing demo

### ISSUE THREAD

**Issue by ricpelo** [godotengine/godot#2941]
Using latest git HEAD, when I open the Screen Capturing demo, I get the following error messages in the console:

```
~/src/godot (master ✔) ᐅ bin/godot.x11.tools.64 
OPENING: /home/ricardo/src/godot/demos/viewport/screen_capture (::home::ricardo::src::godot::demos::viewport::screen_capture)
freed code id 1
~/src/godot (master ✔) ᐅ VBoxContainer: 136,100
ERROR: get_edited_scene_root: Index current_edited_scene out of size (edited_scene.size()).
   At: tools/editor/editor_data.cpp:500.
ERROR: get_edited_scene_root: Index current_edited_scene out of size (edited_scene.size()).
   At: tools/editor/editor_data.cpp:500.
bind toptions
ERROR: get_edited_scene_root: Index current_edited_scene out of size (edited_scene.size()).
   At: tools/editor/editor_data.cpp:500.
ERROR: _printerr: res://screen_capture.gd:2 - Parse Error: Expected '['
   At: scene/resources/scene_format_text.cpp:608.
ERROR: set_current_tab: Index p_current out of size (get_tab_count()).
   At: scene/gui/tabs.cpp:501.
```

**Comment by akien-mga** [comment]
It's not specific to this particular demo, it happens on practical all projects. It's due to the WIP support for the tscn/tres parser, which is currently incomplete, but @reduz is about to push the rest.

**Comment by ricpelo** [comment]
Great then, thank you!

**Comment by akien-mga** [comment]
Fixed by 7aa39b7ca.
